src/components/HeroSection/HeroSection.jsx

```diff
@@ -1,72 +1,90 @@
 import React from "react"
-import useStyles from "./style"
-import { Grid, Typography, Container, Box, Button } from "@material-ui/core"
-import image from "../../images/undraw_celebration_0jvk.svg"
+import { motion } from "framer-motion"
+import PlanetOne from "../../images/1.png"
+import PlanetTwo from "../../images/2.png"
+import PlanetThree from "../../images/3.png"
+import {Section,Container,ColumnLeft,ColumnRight,Image,Button} from './style'
 
 export default function HeroSection() {
-	const classes = useStyles()
+  const fadeLeft = {
+    hidden: { opacity: 0, x: -100 },
+    visible: { opacity: 1, x: 0 },
+  }
 
-	return (
-		<div className={classes.bgColor}>
-			<Container>
-				<Grid
-					spacing={4}
-					container
-					direction="row"
-					justify="center"
-					alignItems="flex-start">
-					<Grid
-						container
-						direction="column"
-						spacing={2}
-						item
-						xs={12}
-						sm={8}
-						md={6}
-						>
-						<Grid item>
-							<Typography className={classes.fontTitleSize}>
-								FindYours reunites people 
-								
-								with Lost & Found property
-							</Typography> 
-						</Grid>
-						<Grid item>
-							<Box color="white">
-								<Typography className={classes.txtFontSize}>
-									FindYours, a one of a kind internet lost and found, which
-									matches users who have lost property to those who have found
-									property
-								</Typography>
-							</Box>
-						</Grid>
-						<Grid container direction="row" justify="space-between" item>
-							<Grid item xs={6}>
-								<Button
-									className={`${classes.btn} ${classes.foundBtn}`}
-									variant="contained" 
-									size="large"
-									>
-									SUBMIT FOUND ITEMS
-								</Button>
-							</Grid>
-							<Grid item xs={6}>
-								<Button
-									className={`${classes.btn} ${classes.lostBtn}`}
-									variant="contained"
-									size="large"
+  return (
+    <Section>
+      <Container>
+        <ColumnLeft>
+          <motion.h1
+            initial={{ opacity: 0 }}
+            animate={{ opacity: 1 }}
+            transition={{ duration: 1 }}
+          >
+            Welcome to FindYours
```

Comment:
Any text should be using i18next translation! 
ex: t('Welcome to FindYours')
This way translation will be easier!